frontend/views/partials/editpage.html

```diff
@@ -0,0 +1,63 @@
+<div class="container">
```

Comment:
@Mernawaseif 
please add section identifier to know what begins where
like
<!--edit form starts here-->
 Html code
<!--edit form ends here-->